scripts/umi_binning.sh

```diff
@@ -509,6 +530,8 @@ $GAWK \
       UMI_NAME=match_umi[s]
       sub("_rc", "", UMI_NAME)
       if(roc[s] == "+"){
+ #CS: I am not entirely clear on the -- > notation here as I could not find that much information about it. Will look more into this. Essentially I think
```

Comment:
`rof_sub_pos_n[UMI_NAME]--` is subtracting 1 from `rof_sub_pos_n[UMI_NAME]` in place, i.e. `rof_sub_pos_n[UMI_NAME] = rof_sub_pos_n[UMI_NAME] - 1`, and then it compares that to see if it's greater than 0.